tensorflow/tools/dockerfiles/assembler.py

```diff
@@ -1,73 +1,140 @@
-# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-# ==============================================================================
-"""Assemble common TF Dockerfiles from many parts.
-
-This script constructs TF's Dockerfiles by aggregating partial
-Dockerfiles. See README.md for usage examples.
+"""Multipurpose TensorFlow Docker Helper.
```

Comment:
We need the license header. Please add it back.

Reply:
Whoops, fixed. I wrote a totally new file and forgot to put this back in.